src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -1214,33 +1214,56 @@ void EnTest3_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList1, Gfx** dL
     }
 }
 
-static TexturePtr sEyeTextures[] = {
-    gKafeiEyesOpenTex,     gKafeiEyesHalfTex,   gKafeiEyesClosedTex,   gKafeiEyesRollRightTex,
-    gKafeiEyesRollLeftTex, gKafeiEyesRollUpTex, gKafeiEyesRollDownTex, object_test3_Tex_006680,
+static TexturePtr sEyeTextures[PLAYER_EYES_MAX] = {
+    gKafeiEyesOpenTex,    // PLAYER_EYES_OPEN
+    gKafeiEyesHalfTex,    // PLAYER_EYES_HALF
+    gKafeiEyesClosedTex,  // PLAYER_EYES_CLOSED
+    gKafeiEyesRightTex,   // PLAYER_EYES_RIGHT
+    gKafeiEyesLeftTex,    // PLAYER_EYES_LEFT
+    gKafeiEyesUpTex,      // PLAYER_EYES_UP
+    gKafeiEyesDownTex,    // PLAYER_EYES_DOWN
+    gKafeiEyesWincingTex, // PLAYER_EYES_WINCING
 };
 
-static TexturePtr sMouthTextures[] = {
-    gKafeiMouthClosedTex,
-    gKafeiMouthTeethTex,
-    gKafeiMouthAngryTex,
-    gKafeiMouthHappyTex,
+static TexturePtr sMouthTextures[PLAYER_MOUTH_MAX] = {
+    gKafeiMouthClosedTex, // PLAYER_MOUTH_CLOSED
+    gKafeiMouthHalfTex,   // PLAYER_MOUTH_HALF
+    gKafeiMouthOpenTex,   // PLAYER_MOUTH_OPEN
+    gKafeiMouthSmileTex,  // PLAYER_MOUTH_SMILE
 };
 
-typedef struct {
-    /* 0x0 */ u8 eyeIndex;
-    /* 0x1 */ u8 mouthIndex;
-} KafeiFace; // size = 0x2
-
-static KafeiFace sFaceExpressions[] = {
-    { 0, 0 }, { 1, 0 }, { 2, 0 }, { 0, 0 }, { 1, 0 }, { 2, 0 }, { 4, 0 }, { 5, 1 }, { 7, 2 }, { 0, 2 },
-    { 3, 0 }, { 4, 0 }, { 2, 2 }, { 1, 1 }, { 0, 2 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 },
+static PlayerFaceIndices sKafeiFaces[PLAYER_FACE_MAX] = {
```

Comment:
Similary, for the face data, `sKafeiFaces` is entirely identical to OoT’s `sPlayerFaces`, but MM’s `sPlayerFaces` has just 1 value changed in the last entry of `sPlayerFaces`.